c.c

```diff
@@ -1927,7 +1933,7 @@ static void processToken (tokenInfo *const token, statementInfo *const st)
 		case KEYWORD_SHORT:     st->declaration = DECL_BASE;            break;
 		case KEYWORD_SIGNED:    st->declaration = DECL_BASE;            break;
 		case KEYWORD_STRING:    st->declaration = DECL_BASE;            break;
-		case KEYWORD_STRUCT:    st->declaration = DECL_STRUCT;          break;
+		case KEYWORD_STRUCT:    checkIsClassEnum (st, DECL_CLASS);      break;
```

Comment:
this should be `DECL_STRUCT`, like it was in Geany (https://github.com/b4n/fishman-ctags/commit/52f09c44690a59aa8b8e6df5e4ebe9df0aae9186, and https://github.com/geany/geany/commit/6c7f69578d8e142f5994cc9cf0e0abc83a606a1b)